src/plugins/ineco/index.js

```diff
@@ -0,0 +1,247 @@
+}
+
+const dateRegexp = /^.*(\d{2}\/\d{2}\/\d{4} \d{2}:\d{2}:\d{2}).*$/
+const currencyConversionRegexp = /^.* ([0-9.]+) ([A-Z]+) \/.* ([0-9.]+) ([A-Z]+).*$/
+const formatDate = date => `${formatNumber(date.getDate())}/${formatNumber(date.getMonth() + 1)}/${date.getFullYear()}`
+const parseAmount = amount => parseFloat(amount.replace(/\s/g, '').replace(/,/g, ''))
+const parseTransactionDate = (details, date) => {
+  const dtls = details.match(dateRegexp)
+
+  if (!dtls) {
+    const dateParts = date.split('/')
+
+    if (dateParts.length >= 3 && dateParts[0] && dateParts[1] && dateParts[2]) {
+      return `${dateParts[2]}-${dateParts[1]}-${dateParts[0]}`
+    }
+
+    return undefined
+  }
+
+  return moment(dtls[1], 'DD/MM/YYYY hh:mm:ss').valueOf()
+}
+
+/**
+ *
+ * @param {AccountHelper} ah
+ * @param account
+ * @param csv
+ * @returns {[]}
+ */
+const parseTransactions = (ah, account, csv) => {
+  const transactions = parseCSV(csv).filter(line => !!line[0] && !isNaN(line[0]))
+  const trans = []
+
+  for (let i = 0; i < transactions.length; i++) {
+    // eslint-disable-next-line no-unused-vars
+    const [nn, id, transactionDate, currency, income, expense, trAccount, payee, details] = transactions[i]
+
+    const tr = {
+      _fromAccount: account.id,
+      id,
+      payee,
+      date: parseTransactionDate(details, transactionDate),
+      comment: details
+    }
+
+    if (income && income !== '.00') {
+      tr.income = parseAmount(income)
+      tr.outcome = 0
+    } else {
+      tr.outcome = parseAmount(expense)
+      tr.income = 0
+    }
+
+    if (tr.income) {
+      tr.incomeAccount = account.id
+
+      if (account.id !== trAccount && ah.has(trAccount)) {
+        tr.outcome = tr.income
+        tr.outcomeAccount = trAccount
+      } else {
+        tr.outcomeAccount = account.id
+      }
+    } else {
+      tr.outcomeAccount = account.id
+
+      if (account.id !== trAccount && ah.has(trAccount)) {
+        tr.income = tr.outcome
+        tr.incomeAccount = trAccount
+      } else {
+        tr.incomeAccount = account.id
+      }
+    }
+
+    if (ah.isDifferentCurrencies(tr.incomeAccount, tr.outcomeAccount)) {
+      // we need to set correct income/outcome when transfer in different currencies
+      // trying to get this info from details
+      const conversion = details.match(currencyConversionRegexp)
+
+      if (conversion) {
+        const incomeCurrency = ah.currency(tr.incomeAccount)
+        const outcomeCurrency = ah.currency(tr.outcomeAccount)
+        const currentCurrency = ah.currency(account.id)
+
+        if (outcomeCurrency === currentCurrency) {
+          tr.income = parseFloat(conversion[3])
+          tr.opOutcome = tr.income
+          tr.opOutcomeInstrument = incomeCurrency
+        } else {
+          tr.outcome = parseFloat(conversion[1])
+          tr.opIncome = tr.outcome
+          tr.opIncomeInstrument = outcomeCurrency
+        }
+      }
+    }
+
+    trans.push(tr)
+  }
+
+  return trans
+}
```

Comment:
It would be great if you place your converting code into converters.js and provide tests for common cases :)

Reply:
moved
will add tests. is it blocks going to prod?)

Reply:
tests are added